### PR TITLE
Add example query for rate calculation

### PR DESCRIPTION
**Pull request by svenklemm**
This adds an example query for rate calculation to the docs.

### CODE REVIEW

File: using-timescaledb/reading-data.md [hunk 1]
```diff
@@ -96,6 +96,29 @@ SELECT time, AVG(temperature) OVER(ORDER BY time
   WHERE location = 'garage' and time > NOW() - interval '1 day'
   ORDER BY time DESC;
 ```
+
+### Rate [](rate)
+
+To calculate the rate for monotonically increasing counters like bytes sent
+of a host or container you need to account for counter resets caused for
```
Comment (solugebefola) [inline using-timescaledb/reading-data.md]: "of" >> "from"?
comma after "caused"

File: using-timescaledb/reading-data.md [hunk 2]
```diff
@@ -96,6 +96,29 @@ SELECT time, AVG(temperature) OVER(ORDER BY time
...
+example by host reboots or container restarts.  The following query calculates
+bytes per second sent and takes counter resets into account.
```
Comment (solugebefola) [inline using-timescaledb/reading-data.md]: "and takes" >> "while taking"?

**Comment by RobAtticus** [comment]
Nit: Can we use the same style for commit messages here as we do main repo? In this case, capitalize 'add'

**Comment by svenklemm** [comment]
I adjusted the wording

**Pull request merged**